### DISCUSSION TITLE
Display the unrounded quantity in the Holding Detail Dialog / Display quantity in Holdings table / Display quantity per account

### DISCUSSION THREAD

**Discussion by swissbuechi** [ghostfolio/ghostfolio#4057] [Ideas]
# Display quantity

Here are some suggestions to improve the visibility of the "quantity" for each holding/account you currently own.

## Holding Detail Dialog show exact quantity
It would be nice if the number representing the "quantity" in the holding detail dialog (`holdingDetailDialog`) would not round to two decimals and instead show the real amount of shares or coins you really own.

![image](https://github.com/user-attachments/assets/7fc11bb3-db99-4da6-9d6c-84aa9d588106)

## Holdings Overview Table add "quantity" row
In addition, it would be awesome to have the quantity as a number in the Holdings overview table (`/home/holdings`).
Don't round quantity here too, to be exact.

![image](https://github.com/user-attachments/assets/dc8caf6a-3e6b-4887-8425-cb8cb7e1be15)

## Per Account quantity
Since you often split up the same holding across multiple accounts/wallets, you would maybe also like to know the quantity of the holding per account. Adding the quantity row to the overview table like suggested above would already solve this, since using an account filter would adjust the amount accordingly.

In addition, it could also be displayed when opening the Account Detail Dialog (`accountDetailDialog`) via `/accounts?accountId=<id>`. Don't round quantity here too, to be exact.

![image](https://github.com/user-attachments/assets/00f1a83b-7c64-4407-ab29-64dec3bd3459)

**Comment by nerdyjan** [comment]
one year after this request it get's more and more complicated with this rounding to two decimals ;(

Think about Crypto, ETF Sparpläne, etc. pp.

rounding mostly sucks in financial applications!

**Comment by amollerup** [comment]
Still waiting for the ability to easily view quantities of specific holdings within a given account (e.g. by applying filters). From my perspective, this feature is crucial to the viability of Ghostfolio.

**Reply by dtslvr** [reply]
This feature will be part of the next release. You can check out the pull request here: https://github.com/ghostfolio/ghostfolio/pull/7595
